sanitizer/_synthesize_analysis.js

```diff
@@ -1,7 +1,15 @@
 const _ = require('lodash');
 
-const fields = ['address', 'neighbourhood', 'borough', 'city', 'county',
-  'state', 'postalcode', 'country'];
+const fields = {
+  'address': 'address',
+  'neighbourhood': 'neighbourhood',
+  'borough': 'borough',
+  'locality': 'city',
```

Comment:
We adopted the libpostal names for our internal representation:

```
> lancaster, pa

Result:

{
  "city": "lancaster",
  "state": "pa"
}
```

This change is just for outward-facing behavior.  People shouldn't be coding against our internal analysis representation but that should be changed at some point, though it's a bigger effort than what's been done here.